framework/source/class/qx/Theme.js

```diff
@@ -321,7 +327,8 @@ qx.Bootstrap.define("qx.Theme",
         "appearances" : "object", // Map
         "meta"        : "object", // Map
         "include"     : "object", // Array
-        "patch"       : "object"  // Array
+        "patch"       : "object", // Array
+        "boot"		  : "function" // Function
```

Comment:
Being anal retentive here... The colons all align, except for this added one. Since there's already another required change, let's align the colons.

Reply:
Yup. Agreed. I took out the tab I had in there and replace with all single spaces like the others